### DISCUSSION TITLE
how can i start new map??

### DISCUSSION THREAD

**Discussion by sheep06031** [Nimdy/Dedicated_Valheim_Server_Script#149] [Q&A] [answered]
I want to start new map with my friend 

but i don't know how to remove origin map and start new map.... 

is there any method??
if i remove origin file, how can i remove origin file while using ssh

**Comment by Nimdy** [comment, marked as answer]
https://github.com/Nimdy/Dedicated_Valheim_Server_Script/wiki/Migrate-Valheim-Map-Data-from-server-to-server

or change your map name and restart valheim services.  The game will auto build you a new map
